sudoku/sudoku.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import dimod
+import math
+import sys
+
+from dimod.generators.constraints import combinations
+from hybrid.reference import KerberosSampler
+
+
+def get_label(row, col, digit):
+    """Returns a string.
+    """
+    return "{row},{col}_{digit}".format(**locals())
+
+
+def get_matrix(filename):
+    """Reads a text file, stores the information in a list of lists, and
+    returns said list of lists.
+
```

Comment:
Something like, "Return a list of lists containing the content of the input text file"?